clients/go-ethereum_master/Dockerfile

```diff
@@ -27,7 +27,9 @@ RUN chmod +x /geth.sh
 
 # Inject the enode id retriever script
 ADD enode.sh /enode.sh
+RUN chmod +x /enode.sh
 
+ADD genesis.json /genesis.json
```

Comment:
It is not needed for Hive, because genesis.json is copied dynamically from the host to the container when Hive is run. But when debugging, I do docker run <container> and then try to connect directly against it. Without that genesis.json as a default, it terminates.